src/common/components/Header/Header.module.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  background: #003928;
```

Comment:
Since we're using SCSS, you can create SCSS variables for theme variables. That way, you can reuse colours throughout the code, but only change them in one place.

To do so, create a `_constants.scss` file in `common/styles` with your SCSS variables:

```
$some-color: #003928;
```

Then, import it into your other SCSS files with: `@import "common/styles/constants";`